resources/developer-mode.mjs

```diff
@@ -42,7 +46,33 @@ export function createUIForMeasurementMethod() {
     return label;
 }
 
-export function createUIForWarmupSuite() {
+
+function createUIForIterationCount() {
```

Comment:
Can we not move this function around? The diff is harder to read because of it.

Reply:
or at least do this in separate commits